src/Microsoft.AspNetCore.Routing/Internal/RequiredValueHelpers.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Routing.Patterns;
+
+namespace Microsoft.AspNetCore.Routing.Internal
+{
+    internal static class RequiredValueHelpers
```

Comment:
no helpers.

Reply:
Is an internal method on RoutePattern an acceptable place?